composer.json

```diff
@@ -21,20 +28,41 @@
     "wp-coding-standards/wpcs": "^2.2"
   },
   "config": {
+    "discard-changes": true,
     "platform": {
       "php": "5.6"
     },
     "sort-packages": true
   },
+  "extra": {
+    "enable-patching": false
+  },
   "autoload": {
     "psr-4": {
-      "Google\\Web_Stories\\": "includes"
-    }
+      "Google\\Web_Stories\\": "includes",
+      "AmpProject\\": "vendor/ampproject/amp-wp/lib/common/src",
+      "AmpProject\\Optimizer\\": "vendor/ampproject/amp-wp/lib/optimizer/src"
+    },
+    "exclude-from-files": [
+      "ampproject/amp-wp/back-compat/back-compat.php",
+      "ampproject/amp-wp/includes/amp-helper-functions.php",
+      "ampproject/amp-wp/includes/admin/functions.php",
+      "ampproject/amp-wp/includes/deprecated.php",
+      "ampproject/amp-wp/docs/includes/register-wp-cli-commands.php"
+    ]
```

Comment:
```suggestion
    }
```

Reply:
Using `exclude-from-files` from the `autoload` property is non-standard in regards to Composer, discouraged by me, and removed in the latest version of the plugin. Sorry for the trouble.